t/author/01-confirm-schema.t

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env perl
```

Comment:
If you put these in an `xt` directory instead of `t` we don't need to set the AUTHOR_TESTING env var.

Reply:
I will try it.

you mean xt/t ?

Reply:
no, just xt